src/main/java/dev/backend/UniTalk/payload/request/RegisterRequest.java

```diff
@@ -0,0 +1,24 @@
+package dev.backend.UniTalk.payload.request;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Getter @Setter
+public class RegisterRequest {
+    @NotBlank
+    @Size(min = 3, max = 20)
+    private String username;
+
+    @NotBlank
+    @Size(max = 50)
+    @Email
+    private String email;
+
+    @NotBlank
+    @Size(min = 6, max = 40)
+    private String password;
+}
```

Comment:
we are probably going to add login at front-end in the next sprint so I suggest adding name and surname fields now